### ISSUE TITLE
Recommend refactor

### ISSUE THREAD

**Issue by mschipperheyn** [spoike/refluxjs-todo#18]
I recommend you refactor your code to work as regular "jsx" to be converted to js. The src kinda looks like a "build" right now. It would make the example more like the rest of us probably build with React and I suspect you might run into some of the same problems we run into with refluxjs getting started.

**Comment by ericmasiello** [comment]
Agreed. That was one of the issues I ran into early on. Fwiw, I found using browserify with the JSX transformer did the trick for me
